### PR TITLE
BaSP-MR-88: Create Global Button

### PR DESCRIPTION
**Pull request by FrancoLelli**
I created a global button

### CODE REVIEW

**Review by Ginoboca1 — APPROVED** [review]
Good job!

File: src/Components/Shared/Button/button.module.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  font-weight: 500;
+  padding: 10px;
+}
+.btnCancel {
+  background: #ffffff;
+  color: #373867;
+  border: 1px solid #373867;
+  cursor: pointer;
+  border-radius: 11.5px;
+  cursor: pointer;
```
Comment (YhanahiRosenthal) [inline src/Components/Shared/Button/button.module.css:18]: you should delete cursor:pointer repeated

**Review by CristianLotorto — APPROVED** [review]
cool buttons!

**Review by YhanahiRosenthal — APPROVED** [review]

**Comment by FrancoLelli** [comment]
[Global Button component](https://trello.com/c/CUipj2d0/88-global-button-component)

**Pull request merged**